lms/djangoapps/badges/api/urls.py

```diff
@@ -0,0 +1,27 @@
+"""
+URLs for badges API
+"""
+from django.conf.urls import patterns, url
+from django.conf import settings
+
+from .views import BadgeClassDetail, UserBadgeAssertions
+from openedx.core.djangoapps.user_api.urls import USERNAME_PATTERN
+
+BADGE_PATTERN = r'(?P<issuing_component>[-\w]*)/(?P<slug>[-\w]+)'
+
+BADGE_PATTERN_COURSE = BADGE_PATTERN + '/' + settings.COURSE_ID_PATTERN
+
+urlpatterns = patterns(
+    'mobile_api.badges.views',
+    url('^classes/' + BADGE_PATTERN + '/$', BadgeClassDetail.as_view(), name='badge_class-detail'),
```

Comment:
Please see the following sections of the [edX REST API Conventions](https://openedx.atlassian.net/wiki/display/AC/edX+REST+API+Conventions:

* #2 rule of **1. URL Naming**:  Keep the API name flat
* "Composite Resource" rule of **2. Identifying Resources**.
* #1 rule of **4. URL Parameters**: sweep complexity behind the `?`.  

These rules need to be applied to this case.